### DISCUSSION TITLE
Advanced Data Types

### DISCUSSION THREAD

**Discussion by huntersite** [atrocore/atrocore#855] [Q&A]
I do not see the option to select "Relationship" when adding an entity. How do I do this?

**Comment by TerminalTitanium** [comment]
<img width="385" alt="Screenshot 2023-11-02 at 17 19 11" src="https://github.com/atrocore/atrocore/assets/31609285/85c25a95-a226-4f93-8e72-1f2fa88d82ce">
How to get this? It isn't in store

**Reply by AnnZast** [reply]
Please send a request to our mail team@atrocore.com to get the module
